### DISCUSSION TITLE
How to host on Rasberry Pi?

### DISCUSSION THREAD

**Discussion by kenyonbowers** [pocketbase/pocketbase#1253] [Q&A]
How easy would it be to host a PocketBase on a Rasberry Pi?

**Comment by ganigeorgiev** [comment]
It shouldn't be much different than any other linux vm.

Depending on the Raspberry PI architecture (32 vs 64bit) and its installed OS you can try the prebuilt ARM binaries:

- armv7 (32bit OS):
  https://github.com/pocketbase/pocketbase/releases/download/v0.9.2/pocketbase_0.9.2_linux_armv7.zip
  
- arm64 (64bit OS):
  https://github.com/pocketbase/pocketbase/releases/download/v0.9.2/pocketbase_0.9.2_linux_arm64.zip

I personally haven't tested it on an arm platform so I cannot fully confirm that it works.
Several users reported that it works as expected for them, others mentioned that they are not able to run the above binaries for some still unknown reason - https://github.com/pocketbase/pocketbase/issues/757.

**Reply by kenyonbowers** [reply]
I see. Thanks for the fast response! I will test them out.

**Reply by j-wil** [reply]
@kenyonbwrs any luck how did it work?

**Reply by kenyonbowers** [reply]
It did work. But I don't think it's the best idea if I am being honest lol

**Reply by scottscharl** [reply]
Why not?

**Reply by kenyonbowers** [reply]
Because, you can't just upgrade RAM as easy.

Edit:
I am comparing to something like a linode server.

**Reply by opencoca** [reply]
Running on a pi is a great option to get up and going. When you want more resources you can move to another platform.

**Comment by altayevrim** [comment]
I'm planning to test it on Raspi 3 with 8gigs of RAM. I'll let you guys know the results.
Also have couple zeros as well. If I'll have time I'll test it on those as well. 

IMO Raspberry Pi would be good fit for small projects or internal management stuff.

**Reply by ganigeorgiev** [reply]
_This is an old discussion and several users confirmed that the ARM builds work fine for them. Just make sure that you use the latest release and the correct ARMv7 or ARM64 build for your device - https://github.com/pocketbase/pocketbase/releases._
